jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -693,6 +697,12 @@ IGN_LAUNCH_COMMON_DEPENDENCIES="libignition-cmake2-dev \\
                          qtquickcontrols2-5-dev \\
                          libqt5core5a"
 
+if [[ -n "${IGN_LAUNCH_MAJOR_VERSION}" && ${IGN_LAUNCH_MAJOR_VERSION} -ge 3 ]]; then
```

Comment:
The dependencies don't match `ign-launch3`

Reply:
ae2782189f180635a8bc3e9d90647023746493f0